lib/install/management/dispatcher.go

```diff
@@ -75,18 +120,16 @@ var diagnosticLogs = make(map[string]*diagnosticLog)
 // NewDispatcher creates a dispatcher that can act upon VIC management operations.
 // clientCert is an optional client certificate to allow interaction with the Docker API for verification
 // force will ignore some errors
-func NewDispatcher(ctx context.Context, s *session.Session, conf *config.VirtualContainerHostConfigSpec, force bool) *Dispatcher {
+func NewDispatcher(ctx context.Context, s *session.Session, action Action, force bool) *Dispatcher {
 	defer trace.End(trace.Begin(""))
 	isVC := s.IsVC()
 	e := &Dispatcher{
+		Action:  action,
 		session: s,
 		op:      trace.FromContext(ctx, "Dispatcher"),
 		isVC:    isVC,
 		force:   force,
 	}
-	if conf != nil {
-		e.InitDiagnosticLogsFromConf(conf)
```

Comment:
Having moved this to create specifically have we lost any functionality from the other actions?

Reply:
No - create was the only call that provided a `conf` -- so the only path that initialized it.  All other calls passed a `nil`